.github/workflows/scripts/pr-tests/.pinot_tests_unit.sh

```diff
@@ -33,6 +33,7 @@ if [ "$RUN_TEST_SET" == "1" ]; then
       -pl 'pinot-spi' \
       -pl 'pinot-segment-spi' \
       -pl 'pinot-common' \
+      -pl ':pinot-yammer' \
```

Comment:
Is this related?

Reply:
yes, the new shade plugin doesn't build the test jar, so need to explicitly specify this here.

Reply:
I don't fully follow. Does the first test set depend on `pinot-yammer`? If so, should we exclude it from the second test set?
What does the leading `:` stand for?

Reply:
yes, `pinot-common` uses `pinot-yammer` as test-jar dependency
`pinot-yammer` is under a path like `pinot-plugins/pinot-metrics/pinot-yammer`, so using `:pinot-yammer` will match the module path.

Reply:
In that case, should we move the `pinot-metrics` tests into this test set and exclude it from the second test set?

Reply:
It's a bit complicated as we need to put all the submodules of `pinot-metrics` into .
Also this `pinot-yammer` is a very small test, so it's ok to run it double times.